plugins/stylelint.cjs

```diff
@@ -1,69 +1,5 @@
-const { tokens } = require('@rhds/tokens');
+const rules = require('./stylelint/rules.cjs');
 const stylelint = require('stylelint');
-const parser = require('postcss-value-parser');
 
-const ruleName = 'rhds/token-values';
-
-const messages = stylelint.utils.ruleMessages(ruleName, {
-  expected: 'Expected ...'
-});
-
-const meta = {
-  url: 'https://github.com/RedHat-UX/red-hat-design-system/tree/main/plugins/token-values.js',
-};
-
-/** @type {import('stylelint').Plugin} */
-const ruleFunction = (_options, _secondaryOptions, ctx) => {
-  return (root, result) => {
-    const validOptions = stylelint.utils.validateOptions(
-      result,
-      ruleName,
-      {
-        /* .. */
```

Comment:
Any reason to keep these comment lines?  I noticed they are in the plugin documentation but seem like placeholders for optional values we could remove.